sim-lib/src/lnd.rs

```diff
@@ -88,6 +81,28 @@ impl LightningNode for LndNode {
         &self.info
     }
 
+    async fn get_network(&mut self) -> Result<Network, LightningError> {
+        let info = self
+            .client
+            .lightning()
+            .get_info(GetInfoRequest {})
+            .await
+            .map_err(|err| LightningError::GetInfoError(err.to_string()))?
+            .into_inner();
+
+        if info.chains.is_empty() {
```

Comment:
Could tighten this to check `len=1` because _theoretically this API allows multiple chains (I _believe_ there was a time where you could run litecoin and bitcoin, good god whyyy). I don't think we'll ever hit it but it's strictly the most correct way to handle it.

Reply:
So if they are connected to more than one backend we error out?

Reply:
Yeah, just enforce one chain to keep it simple.

Reply:
Also like ... be normal. GTFO with yo litecoin lightning.